.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,58 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+env:
+  DBT_PROFILES_DIR: ./
+  DBT_VERSION: 0.21.0
+
+
+defaults:
+  run:
+    working-directory: ./integration_tests
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  pull_request:
+    branches: [ master ]
```

Comment:
Unrelated to this PR: Can we rename the branch to `main`? 😇

Reply:
for sure we should!